src/main/java/org/commonjava/util/sidecar/services/ProxyService.java

```diff
@@ -103,28 +104,32 @@ void handleConfigChange( String message )
         logger.debug( "Handle event {}, refresh timeout: {}", EVENT_PROXY_CONFIG_CHANGE, timeout );
     }
 
+    public Uni<Response> doHead( String packageType, String type, String name, String path, HttpServerRequest request )
+                    throws Exception
+    {
+        String contentPath = UrlUtils.buildUrl( CONTENT_REST_BASE_PATH, packageType, type, name, path );
+        return doHead( contentPath, request );
+    }
+
     @GET
```

Comment:
I think these GET annotations should be removed from service class.

Reply:
@ligangty committed, done